dff/script/import_export/parser/cli.py

```diff
@@ -0,0 +1,119 @@
+"""This module contains cli wrappers around parser tools
+"""
+from pathlib import Path
+import argparse
+from dff.script.import_export.parser.tools import py2yaml, yaml2py, py2graph, graph2py
+
+
+def is_dir(arg: str) -> Path:
+    """Check that the passed argument is a directory
+
+    :param arg: Argument to check
+    :type arg: str
```

Comment:
Arguments will be set automatically (or at least they should be)

Reply:
Could you elaborate?
Set automatically to what? By which entity?